docs/installation_mac-os-x.md

```diff
@@ -0,0 +1,365 @@
+
+
+### Install Composer
+
+MFTF requires Composer 1.3 or later.
+
+#### Download the composer installer
+
+```bash
+php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+```
+
+#### Verify composer installer
+
+```bash
+php -r "if (hash_file('sha384', 'composer-setup.php') === 'a5c698ffe4b8e849a443b120cd5ba38043260d5c4023dbf93e1558871f1f07f58274fc6f4c93bcfd858c6bd0775cd8d1') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;"
+```
+
+#### Install composer globally
+
+To install to /usr/local/bin. enter:
+
+```bash
+php composer-setup.php --install-dir=/usr/local/bin --filename=composer
+```
+
+### Install Docker
+
+If you want have Docker previously installed, download docker for Mac OS X from [stable channel][docker].
+
+## Install Magento {#install-magento}
+
+Follow Magento Installation Guide to install Magento either by [Git][magento_install_git] clone or by [Composer][magento_install_composer].
+
+## Prepare Magento  {#prepare-magento}
+
+Configure the following settings in Magento as described below.
+
+### WYSIWYG settings    {#wysiwyg-settings}
+
+A Selenium web driver cannot enter data to fields with WYSIWYG.
+
+To disable the WYSIWYG and enable the web driver to process these fields as simple text areas:
+
+1. Log in to the Magento Admin as an administrator.
+2. Navigate to **Stores** > Settings > **Configuration** > **General** > **Content Management**.
+3. In the WYSIWYG Options section set the **Enable WYSIWYG Editor** option to **Disabled Completely**.
+4. Click **Save Config**.
+
+<div class="bs-callout bs-callout-tip">
+When you want to test the WYSIWYG functionality, re-enable WYSIWYG in your test suite.
+</div>
+
+### Security settings   {#security-settings}
+
+To enable the **Admin Account Sharing** setting, to avoid unpredictable logout during a testing session, and disable the **Add Secret Key in URLs** setting, to open pages using direct URLs:
+
+1. Navigate to **Stores** > Settings > **Configuration** > **Advanced** > **Admin** > **Security**.
+2. Set **Admin Account Sharing** to **Yes**.
+3. Set **Add Secret Key to URLs** to **No**.
+4. Click **Save Config**.
+
+### Nginx settings {#nginx-settings}
+
+If Nginx Web server is used on your development environment then **Use Web Server Rewrites** setting in **Stores** > Settings > **Configuration** > **Web** > **Search Engine Optimization** must be set to **Yes**.
+
+To be able to run Magento command line commands in tests add the following location block to Nginx configuration file:
+
+```conf
+location ~* ^/dev/tests/acceptance/utils($|/) {
+  root $MAGE_ROOT;
+  location ~ ^/dev/tests/acceptance/utils/command.php {
+      fastcgi_pass   fastcgi_backend;
+      fastcgi_index  index.php;
+      fastcgi_param  SCRIPT_FILENAME  $document_root$fastcgi_script_name;
+      include        fastcgi_params;
+  }
+}
+```
+
```

Comment:
I wouldn't add this as `Install step`
I would specify in prerequisite that Magento 2 instance is available by url (magento.local or something) on your local machine.
And I would have all of Magento configuration in another document called `Prepare Magento 2 for MFTF testing`

Reply:
Fixed. 
Magento 2 is now in prerequisite and add a separate page for magento configuration.